mods/ra/rules/vehicles.yaml

```diff
@@ -733,6 +736,9 @@ QTNK:
 		RevealGeneratedShroud: False
 	RevealsShroud@GAPGEN:
 		Range: 4c0
+	RevealOnDeath:
```

Comment:
We could modify the `MadTank` trait (and also `AttackSuicides` for the demo truck) to grant a condition immediately before they die.  This could be used to enable the trait (and possibly other effects?) only when they are deliberately detonated.